core-services/prow/02_config/openshift/origin/_pluginconfig.yaml

```diff
@@ -1,3 +1,55 @@
+label:
+  restricted_labels:
+    openshift/origin:
+    - allowed_users:
+      - LiliC
```

Comment:
Since I am going, not sure I need to be added here and then removed later unless it's somehow automated?